.circleci/scripts/check_for_casetwins.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+shopt -s globstar nullglob
+
+# We scan the Poplog tree looking for ordinary files whose paths differ
+# only by case e.g. 
+#   pop/packages/vedmail/auto/ved_respond.p
+#   pop/packages/vedmail/auto/ved_Respond.p
+# These files will cause difficulties if we ever move to a filing system
+# that is case-preserving but case-insensitive (Windows, MacOS HFS).
+# We count this as techdebt.
+
+# We have a list of known problems and we want to be warned if we find
+# that list is expanded.
+
+mkdir -p _build/poplog_base/
```

Comment:
I don't think a checking script should make any permanent directories, especially the build folder. I'd check the folder exists and exit if it doesn't